src/i18n/content/jp/docs/new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx

```diff
@@ -17,15 +17,16 @@ redirects:
   - /docs/new-relic-one/use-new-relic-one/core-concepts/explorer-view-performance-across-apps-services-hosts
   - /docs/new-relic-solutions/best-practices-guides/full-stack-observability/monitor-apps-hosts-health-maps/
   - /docs/new-relic-one-new-relic-explorer
+  - /docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts
 ---
 
-In digital business, systems are becoming increasingly large, complex, and interdependent. You have hundreds of applications and services running at the same time, and you need to monitor thousands of elements emitting data (we call those data sources [entities](/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic)).
+In digital business, systems are becoming increasingly large, complex, and interdependent. You have hundreds of applications and services running at the same time, and you need to monitor thousands of elements emitting data. In New Relic, we call all those data sources [entities](/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic).
 
-With the New Relic Explorer, we give you a new, efficient front-door experience to easily observe the full stack of your software. We automatically create and maintain visibility from the data you send us. Use the New Relic Explorer to access and give context to the performance data from all your monitored applications, services, hosts, containers, synthetic monitors, etc. You can quickly see the entities related to a problem, exposing possible root causes and what other systems might be affected.
+The New Relic Explorer it's our new, efficient front-door experience to easily observe the full stack of your software. We automatically create and maintain visibility from the data you send us. Use the New Relic Explorer to access and give context to the performance data from all your monitored applications, services, hosts, containers, synthetic monitors, etc. You can quickly see the entities related to a problem, exposing possible root causes and what other systems might be affected. Want to try it? [Create a New Relic account](https://newrelic.com/signup) for free, forever!
```

Comment:
Explorer is no longer new, plus there's a missing verb. Maybe reword to "The New Relic Explorer gives you an efficient, front-door experience" etc

Reply:
Yep!